app/services/orphaned_profiles_cleaner.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Service for cleaning up orphaned profiles
+class OrphanedProfilesCleaner
+  class << self
+    def run!
+      Profile.canonical(false).where(policy_id: nil, external: true).destroy_all
```

Comment:
Are there any Profiles that have `external: false` but do not have any policy assigned?

Reply:
No